tests/libcxx/expected_results.txt

```diff
@@ -482,26 +483,15 @@ std/language.support/support.limits/support.limits.general/iterator.version.pass
-std/language.support/support.limits/support.limits.general/functional.version.pass.cpp FAIL
-std/language.support/support.limits/support.limits.general/iterator.version.pass.cpp FAIL
-std/language.support/support.limits/support.limits.general/string_view.version.pass.cpp FAIL
-std/language.support/support.limits/support.limits.general/tuple.version.pass.cpp FAIL
-std/language.support/support.limits/support.limits.general/utility.version.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.char/assign3.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.char/copy.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.char/move.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.char16_t/assign3.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.char16_t/copy.pass.cpp FAIL
+std/strings/char.traits/char.traits.specializations/char.traits.specializations.char/copy.pass.cpp:0 FAIL
+std/strings/char.traits/char.traits.specializations/char.traits.specializations.char/move.pass.cpp:0 FAIL
+std/strings/char.traits/char.traits.specializations/char.traits.specializations.char16_t/copy.pass.cpp:0 FAIL
 std/strings/char.traits/char.traits.specializations/char.traits.specializations.char16_t/move.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.char32_t/assign3.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.char32_t/copy.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.char32_t/move.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.char8_t/assign3.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.char8_t/copy.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.char8_t/move.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.wchar.t/assign3.pass.cpp FAIL
-std/strings/char.traits/char.traits.specializations/char.traits.specializations.wchar.t/copy.pass.cpp FAIL
+std/strings/char.traits/char.traits.specializations/char.traits.specializations.char32_t/copy.pass.cpp:0 FAIL
+std/strings/char.traits/char.traits.specializations/char.traits.specializations.char32_t/move.pass.cpp:0 FAIL
+std/strings/char.traits/char.traits.specializations/char.traits.specializations.char8_t/copy.pass.cpp:0 FAIL
+std/strings/char.traits/char.traits.specializations/char.traits.specializations.char8_t/move.pass.cpp:0 FAIL
+std/strings/char.traits/char.traits.specializations/char.traits.specializations.wchar.t/copy.pass.cpp:0 FAIL
 std/strings/char.traits/char.traits.specializations/char.traits.specializations.wchar.t/move.pass.cpp FAIL

```

Comment:
They fail because one compiler supports constexpr and the other not as far as I understand it.

At least that was what I read from the error message

Reply:
Yeah, the `:0` are MSVC (while `:1` is Clang, due the order in which we list their configurations). Definitely caused by the builtins.

Reply:
Ok, I completely understand what was happening here - the `char16_t` and `wchar_t` `move` tests were failing for **both** MSVC (lacking the builtin) and Clang (because `reinterpret_cast` was being used, which is forbidden for `constexpr`). I should be able to fix all failures now.